frontend/src/metabase/lib/dom.js

```diff
@@ -208,3 +208,10 @@ export function constrainToScreen(element, direction, padding) {
     }
     return false;
 }
+
+// Used for tackling Safari rendering issues
```

Comment:
If you found a page describing this problem and solution it would be good to link to it.